tests/Oryx.BuildImage.Tests/Python/PythonSampleAppsTest.cs

```diff
@@ -1226,13 +1226,19 @@ public void Django_CollectStaticFailure_DoesNotFailBuild()
                 CommandToExecuteOnRun = "/bin/bash",
                 CommandArguments = new[] { "-c", script }
             });
+            // Regex will match:
+            // "yyyy-mm-dd hh:mm:ss"|ERROR|Failed pip installation with exit code: 1
+            // Example:
+            // "2021-10-27 07:00:00"|ERROR|Failed to pip installation with exit code: 1
+            Regex regex = new Regex(@"""[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1]) (0[0-9]|1[0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])""\|WARNING\|.*");
```

Comment:
I hope I'm not missing something, but are we only handling logs with WARNING here?

Reply:
Specifically, given the Assert.Contains we're fixing, shouldn't we be looking for something like "'collectstatic' exited with exit code 1."?

Reply:
The ask changed so the stacktrace of the failure would show; to give better reason of the failure for the user. `collectstatic exited with exit code 1` was too ambiguous. 

However I did update the regex to pattern match for `Exit code 1`